rpc/openapi/openapi.yaml

```diff
@@ -1002,6 +1002,92 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  /prove_shares:
```

Comment:
[Suggestion] As a reviewer, verifying the OpenAPI specifications can be challenging and error-prone. I recommend using tools like [go-swagger](https://github.com/go-swagger/go-swagger) to generate the specifications automatically hence simplifying the process for developers and reviewers, providing greater confidence in the generated code.

Reply:
I am not sure whether the openapi specs in core was generated using go-swagger. We should investigate which tool they used.

For verifying that the specs are valid, I usually use https://editor.swagger.io/ and test the methods myself using an RPC endpoint.